project/MultiScalaProject.scala

```diff
@@ -44,6 +44,7 @@ trait MultiScala {
         val sourceDir = (ThisBuild / baseDirectory).value / base / "src" / "main"
         CrossVersion.partialVersion(scalaVersion.value) match {
           case Some((2, n)) if n >= 12 => List(sourceDir / "scala-2.12+")
+          case Some((3, _))            => List(sourceDir / "scala-2.13")
           case _                       => Nil
         }
       },
```

Comment:
I don't believe this change is necessary.

In fact, it's not clear to me that the rest of this block is required as I don't see a `scala-2.12+` directory anywhere.

Reply:
There is some extra complexity in the build because the repo includes Scalafix rules. I don't really understand that complexity, and perhaps some of it is unnecessary, but I don't think either you nor I needs to worry about it right now, if tests are passing and the community build likes it then :+1:, and the next time we publish if it goes well then :+1: